### ISSUE TITLE
Optimize RSF Hyperparameters

### ISSUE THREAD

**Issue by alishasrivas** [alishasrivas/Predicting-YCStartup-Peformance#34]
Use grid search or random search to tune RSF hyperparameters.
Focus on optimizing the Concordance Index (C-Index) on the validation set.
